File: 2.0/python/calls/findCallsPharmacyReminders.py
from callfire.client import CallfireClient

client = CallfireClient('api-login', 'api-password')
response = client.calls.findCalls(
    # return calls from particular campaign
    campaignId=10,
    label='reminders',
    # time is in unix milliseconds
    intervalBegin=1448920800000,
    intervalEnd=1449698400000,
    limit=10
).result()

# see sample JSON response for this API
# on 'curl' samples tab
print(response)

File: 2.0/php/calls/findCallsPharmacyReminders.php
<?php

class ApiClientSample{

    public static function main(){
        $client = \CallFire\Api\DocumentedClient::createClient("login", "password");
        // find all calls made through campaign with id = 10, label = reminders is assigned to such campaign,
        // between 2015-12-01 and 2015-12-10, return first 10 records if any
        $request = $client->findCalls();
        $request->getOperationConfig()->setQueryParameters(array("campaignId" => 10,
                                                                 "label" => "reminders",
                                                                 "intervalBegin" => 1448920800000,
                                                                 "intervalEnd" => 1449698400000,
                                                                 "limit" => 10));
        $result = $client->request($request);
        $json = json_decode($result->getBody());
    }
}

ApiClientSample::main();
